pulsarbot/entrypoint.sh

```diff
@@ -37,11 +37,11 @@ fi
 PR_NUM=$(jq -r '.issue.number' "${GITHUB_EVENT_PATH}")
 
 # get head sha
-curl -s -H "Accept: application/vnd.github.antiope-preview+json" https://api.github.com/repos/apache/pulsar/git/ref/pull/${PR_NUM}/head > result-headsha.txt
+curl -s -H "Accept: application/vnd.github.antiope-preview+json" "https://api.github.com/repos/${GITHUB_REPOSITORY}/git/ref/pull/${PR_NUM}/head" > result-headsha.txt
```

Comment:
We need to add `apache/pulsar` as the default value otherwise this will break pulsar CI.

Reply:
@zymap thanks for your review, added `BOT_TARGET_REPOSITORY` with default value `apache/pulsar`, PTAL.